promoted_python_delivery_client/model/client_info.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass, field
+from dataclasses_json import config, dataclass_json, LetterCase
+from typing import Optional
+from promoted_python_delivery_client.model.client_type import ClientType
+
+from promoted_python_delivery_client.model.traffic_type import TrafficType
+
+
+@dataclass_json(letter_case=LetterCase.CAMEL)
+@dataclass
+class ClientInfo:
+    client_type: Optional[ClientType] = field(default=None, metadata=config(exclude=lambda v: v is None))  # type: ignore
```

Comment:
Interesting syntax.  I don't know what it means.  I think a bunch of some of the other models have optional fields.

Reply:
field is standard for annotating in dataclasses. The extra metadata in this case is used by dataclasses_json to strip optional fields from the JSON, or else the request json will be full of wasteful nulls (which is what most JSON-marshaling frameworks I looked at for Python did, with pretty mediocre ways of getting around it).